src/Sulu/Bundle/SnippetBundle/Controller/SnippetController.php

```diff
@@ -117,7 +123,8 @@ public function __construct(
         FormFactory $formFactory,
         RequestHashChecker $requestHashChecker,
         ListRestHelper $listRestHelper,
-        MetadataFactoryInterface $metadataFactory
+        MetadataFactoryInterface $metadataFactory,
+        TranslatorInterface $translator
```

Comment:
Should we document BC Breaks in constructors/services definitions @sulu/core-team?

Reply:
That's the reason I would not do it. It is very unlikely that this is done, and if we start to document BC breaks on that level, then the UPGRADE.md file gets very bloated.